CCM-8413: new proofing content

emmagifford opened the pull request:
<!-- markdownlint-disable-next-line first-line-heading -->
## Description

Changes are as follows:

- new page on proofing, /using-nhs-notify/approve-your-messages, positioned second from last in the 'Sending a message' subsection 
- moved /message-channel-supplier-status to 7th in the nav order in 'Sending a message' to make room for the new page

The new content is split into general proofing guidance that applies to all channels, and channel-specific guidance on how to approve or reject proofs, divided into digital channels and letter.

## Context

Our public documentation is missing information about the proofing process. 

Users don't know how our proofing process works, and as a result, they make assumptions to fill in those gaps, which can lead to wasted effort on their part. Our onboarding team have to correct their assumptions.

We need to show users evidence of what their templates look like in their final form for them to sign them off.

Proofing will be incorporated into the UI over time, starting with letter proofing. This content can be updated to align with that.

Had an initial session with Romy Bos to define what the problem is, what actions are needed and what outcomes we want to see. 

The content in Confluence has been reviewed by Romy, Gaby Kenton and Josh Morris. Sommer Croft and Karen Craddock also reviewed the paragraph beginning 'It's your responsibility to review the proof messages...' following a question from Josh about whether to explicitly mention getting clinical sign off.


## Type of changes

<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->

- [x] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->

- [x] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

jakecosgrove commented on docs/pages/using-nhs-notify/approve-your-messages.md:
```diff
@@ -0,0 +1,66 @@
+
+We will not send actual messages or letters directly to you as part of the proofing process.
+
+It's your responsibility to review the proof messages in a way that meets your organisation's needs. This includes making sure you've got approvals from the relevant stakeholders in your team. Compare your proofs against the final templates you submitted to us.
```
The 'compare your proofs' sentence seems out of place. I also wonder if we're leaving it too ambiguous about what they should be proofing by saying it's to their needs. 

I think the only thing NHS Notify is concerned about is that the proof the user receives does not match the template that the user provided. 

If they need to make further changes, they'll need to create a new template. 

One we can discuss so we align with the Web UI letter proofing designs?

emmagifford replied:
These issues have been resolved with the restructure of the content following our 2i discussions - thank you!

jakecosgrove commented on docs/pages/using-nhs-notify/approve-your-messages.md:
```diff
@@ -0,0 +1,66 @@
+
+We'll ask for the contact details of the person in your team who'll approve the proof message.
+
+We'll email you with a screenshot or attachment of your proof message.
```
Maybe: "We'll **then** email..." just as it's a sequential step from the previous sentence.

emmagifford replied:
Much better, thanks.

jakecosgrove commented on docs/pages/using-nhs-notify/approve-your-messages.md:
```diff
@@ -0,0 +1,66 @@
+
+### Approving your proof
+
+Reply to our email to approve your proof message.
```
I wonder if this could be read as sending any sort of reply will approve the proof.

Maybe:
"Reply to our email and tell us that you approve the proof message."

emmagifford replied:
Good spot - updated.

jakecosgrove commented on docs/pages/using-nhs-notify/approve-your-messages.md:
```diff
@@ -0,0 +1,66 @@
+
+### If you need to reject your proof
+
+Reply to our email if the content or formatting does not match the template you submitted. We'll update the proof message and email it to you again.
```
I wonder if we need to explicitly say that they should describe what's wrong with the proof

emmagifford replied:
Good idea - I've added in this detail.

jakecosgrove commented on docs/pages/using-nhs-notify/approve-your-messages.md:
```diff
@@ -0,0 +1,66 @@
+
+Reply to our email if the content or formatting does not match the template you submitted. We'll update the proof message and email it to you again.
+
+If you need to change the content or formatting of your original template, send us an updated [NHS App message]({% link pages/using-nhs-notify/nhs-app-templates.md %}), [email]({% link pages/using-nhs-notify/email-templates.md %}) or [text message]({% link pages/using-nhs-notify/text-message-templates.md %}) Microsoft Word template and start the proofing process again.
```
Would it be clearer if we said they need to update the NHS App/ email/ text word template, then send a new version to replace the existing which will be proofed again?

Maybe:

"If you need to make further changes to your original template, you'll need to update your original word template and send it to us again so we can create a new proof."

emmagifford replied:
Good idea - sorted.

jakecosgrove commented on docs/pages/using-nhs-notify/approve-your-messages.md:
```diff
@@ -0,0 +1,66 @@
+If you need to change the content or formatting of your original template, send us an updated [NHS App message]({% link pages/using-nhs-notify/nhs-app-templates.md %}), [email]({% link pages/using-nhs-notify/email-templates.md %}) or [text message]({% link pages/using-nhs-notify/text-message-templates.md %}) Microsoft Word template and start the proofing process again.
+
+## Letter proofing
+
```
Apologies this is not helpful, but I'm now wondering if we need to separate letter proofing from digital channel proofing at all as the difference seem to only be:

- the users sends us their letter as a PDF instead of a word doc
- they will receive their proof as a PDF rather than a screenshot
- if they send an updated version of their template with changes, it must be a PDF (same as first point?)

emmagifford replied:
These issues have been resolved with the restructure of the content following our 2i discussions - thank you!

jakecosgrove commented on docs/pages/using-nhs-notify/approve-your-messages.md:
```diff
@@ -0,0 +1,66 @@
+
+Find out how to approve or reject your proofs for:
+
+- [NHS App messages, emails and text messages](#nhs-app-messages-emails-and-text-message-proofing)<!-- markdownlint-disable-line -->
```
The accessibility checker does not like this anchor, might be because the heading it's referring to has a comma in it where the anchor link target doesn't?

emmagifford commented:
Further to Jake's (very helpful!) 2i feedback, I have:

- restructured /using-nhs-notify/approve-your-messages, which Romy has now approved
- changed the language about template submission from 'submit' to 'provide' - Alex felt that we should reserve 'submit' for the specific process in the web UI
- added guidance on how users provide their templates as part of the current process in the 4 template pages - we realised that this information was missing

I have also:

- added numbered step overviews at the start of the digital template pages and tweaked the headings, to be consistent with the letter template page
- fixed some copy and paste errors on the template pages
- updated the 'Telling recipients who your XYZ is from' sections on the template pages
